core/lib/config/src/configs/eth_sender.rs

```diff
@@ -3,21 +3,25 @@ use std::time::Duration;
 use serde::Deserialize;
 use zksync_basic_types::H256;
 
+use crate::ETHWatchConfig;
+
 /// Configuration for the Ethereum sender crate.
 #[derive(Debug, Deserialize, Clone, PartialEq)]
-pub struct ETHSenderConfig {
+pub struct ETHConfig {
     /// Options related to the Ethereum sender directly.
-    pub sender: SenderConfig,
+    pub sender: Option<SenderConfig>,
     /// Options related to the `GasAdjuster` submodule.
-    pub gas_adjuster: GasAdjusterConfig,
+    pub gas_adjuster: Option<GasAdjusterConfig>,
+    pub watcher: Option<ETHWatchConfig>,
```

Comment:
Not sure if mixing all of these is a good concept, but no big objections either.